lib/coinbase/transfer.rb

```diff
@@ -185,5 +190,18 @@ def to_s
     def inspect
       to_s
     end
+
+    private
+
+    # Updates the Transfer model with the latest data.
+    def update_model
```

Comment:
Where does this get called? I see this pattern in `Coinbase::Wallet` getting called in the initializer:
https://github.com/coinbase/coinbase-sdk-ruby/blob/2b557e9492f845ec39681ad12ed9600375c0270a/lib/coinbase/wallet.rb#L42

But was curious the intent here

Reply:
Eventually we will use it for updating the status but I didn't want to scope that into this PR. This PR mainly adds the list and get for transfers.

Reply:
Can we just add it when it gets utilized? Ideally we don't have unused private methods and they're fully tested via the public interface

Reply:
yes will remove now

Reply:
done in most recent commit